cli/cmd/root.go

```diff
@@ -90,6 +91,7 @@ var RootCmd = &cobra.Command{
 
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&controlPlaneNamespace, "linkerd-namespace", "l", defaultNamespace, "Namespace in which Linkerd is installed [$LINKERD_NAMESPACE]")
+	RootCmd.PersistentFlags().StringVar(&clusterDomain, "cluster-domain", defaultClusterDomain, "Set custom cluster domain")
```

Comment:
@ihcsim thanks for the suggestion. Having it as a root cmd was nice when i started out because i really liked (still do :stuck_out_tongue: )  the idea of being able to generate service profiles for a different cluster than what's in your kubeconfig.

Since that's no longer in there, there's no reason to keep it in `root.go`. I applied your changes.